scala/scala.bzl

```diff
@@ -522,7 +522,7 @@ def _scala_test_impl(ctx):
     (cjars, rjars) = (jars.compiletime, jars.runtime)
     # _scalatest is an http_jar, so its compile jar is run through ijar
     # however, contains macros, so need to handle separately
-    scalatest_jars = _collect_jars([ctx.attr._scalatest]).runtime
+    scalatest_jars = _collect_jars([ctx.attr._scalatest, ctx.attr._scalatest_runner]).runtime
```

Comment:
Can you see if this works when adding it to the extra_runtime_deps list on line 520? (with _scalatest_reporter)